src/poetry/utils/env/env_manager.py

```diff
@@ -121,11 +160,20 @@ def in_project_venv(self) -> Path:
         venv: Path = self._poetry.file.path.parent / ".venv"
         return venv
 
-    def activate(self, python: str) -> Env:
+    @cached_property
+    def envs_file(self) -> EnvsFile:
         venv_path = self._poetry.config.virtualenvs_path
-        cwd = self._poetry.file.path.parent
+        return EnvsFile(venv_path / self.ENVS_FILE)
```

Comment:
Could we just use the `self._poetry.config.virtualenvs_path` instead of adding a variable?

Reply:
Done.